src/main/java/com/manardenza/dao/HotelDaoImpl.java

```diff
@@ -2,13 +2,18 @@
 
 import com.manardenza.entity.Hotel;
 
+import java.io.File;
 import java.util.ArrayList;
 import java.util.List;
 
 public class HotelDaoImpl extends AbstractDao<Hotel> {
 
     private static HotelDaoImpl instance;
 
+    {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Move the contents of this initializer to a standard constructor or to field initializers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1171)